### PR TITLE
feat(storybooks): Switch Storybook notification from comment post to …

### LINKED ISSUE #5474: Switch Storybook builds from PR comments to a Status Check?
Storybook builds are noisy in comments. This is mainly because it's simple to post a comment and I already knew how to do it 😅 

We could research & implement [GitHub Status Checks](https://developer.github.com/v3/repos/statuses/) so that Storybook builds appear in the status block at the bottom along with other things like CircleCI status. Seems like creating a status is as easy as a comment, but I've never tried it before

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXA-1996)
┆Issue Number: FXA-1996

### PR DESCRIPTION
**Pull request by lmorchard**
fixes #5474

Realized after a skip of the API docs that this might be a super-quick change, so I figured I'd just try it.

### CODE REVIEW

**Comment by lmorchard** [comment]
Okay, let's see if folks like this tweak. No more PR comments. Just a status check at the bottom of the PR. Click "Show all checks" to see this with a link to the build:

Capture

File: packages/fxa-payments-server/src/index.tsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ import { actions } from './store/actions';
 
 async function init() {
   readConfigFromMeta(headQuerySelector);
+
```
Comment (lmorchard) [inline packages/fxa-payments-server/src/index.tsx:17]: Added a whitespace change to payments-server to cause a Storybook build. I can always back this out

File: _scripts/build-storybooks.sh [hunk 2]
```diff
@@ -105,13 +105,16 @@ if [[ ! -z $CI_PULL_REQUEST ]]; then
...
-        https://api.github.com/repos/${PROJECT_REPO}/issues/${PR_NUMBER}/comments;
+        --data "{\"state\":\"success\",\"target_url\":\"$PR_URL\",\"context\":\"storybooks: pull request\",\"description\":\"Storybook deployment for Pull Request\"}" \
+        https://api.github.com/repos/${PROJECT_REPO}/statuses/${COMMIT_HASH};
 fi
 
 echo "Cleaning up $PUBLISH_ROOT"
 cd ..
 rm -rf $PUBLISH_ROOT
+
+0d2a28b2412479f9e8fb07f8a1a2666784957f3a
```
Comment (LZoog) [inline _scripts/build-storybooks.sh:120]: What's this doing?

Comment (lmorchard) [reply _scripts/build-storybooks.sh]: Uhh, me being dumb is what that's doing 😅

**Pull request merged**